dist/setup/index.js

```diff
@@ -135074,6 +135074,9 @@ function getVersionFromFileContent(content, distributionName, versionFile) {
         javaVersionRegExp =
             /^(java\s+)(?:\S*-)?v?(?<version>(\d+)(\.\d+)?(\.\d+)?(\+\d+)?(-ea(\.\d+)?)?)$/m;
     }
+    else if (versionFileName == '.sdkmanrc') {
+        javaVersionRegExp = /^java\s*=\s*(?<version>[^-]+)/m;
```

Comment:
`.sdkmanrc` might exist but describe other software instead of Java. Should the setup script fallback to the next version determination method in such case?